app-admin/cdist/cdist-6.5.4.ebuild

```diff
@@ -0,0 +1,24 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+PYTHON_COMPAT=( python{3_6,3_7} )
+
+inherit distutils-r1
+
+DESCRIPTION="A usable configuration management system"
+HOMEPAGE="https://www.nico.schottelius.org/software/cdist/"
```

Comment:
Please also add https://code.ungleich.ch/ungleich-public/cdist as 2ndary HOMEPAGE. Also pypi page could be added here.

Reply:
Since https://www.nico.schottelius.org/software/cdist/ redirects to https://www.cdi.st/, how about `HOMEPAGE="https://www.cdi.st/ https://code.ungleich.ch/ungleich-public/cdist"`?

Reply:
Currently pushed `HOMEPAGE="https://www.cdi.st/ https://code.ungleich.ch/ungleich-public/cdist"`